x/slashing/client/rest/query.go

```diff
@@ -81,7 +76,7 @@ func signingInfoHandlerListFn(cliCtx context.CLIContext, storeName string, cdc *
 		}
 
 		if len(validators.Validators) == 0 {
-			w.WriteHeader(http.StatusNoContent)
+			w.WriteHeader(http.StatusOK)
```

Comment:
thanks ! we need to also test here that we're returning an empty array `[]` instead of `null`

Reply:
I added a test `TestAccountBalanceQuery` which covers checking if we are returning []